src/Apps/Artwork/Components/ArtworkDetails/ArtworkDetailsArticles.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box } from "@artsy/palette"
+import React from "react"
+
+interface ArtworkDetailsArticlesArtwork {
+  readonly articles: ReadonlyArray<{
+    readonly title?: string
+    readonly href?: string
+    readonly thumbnail?: {
+      readonly image?: {
+        readonly width: number
+        readonly height: number
+        readonly url: string
+      }
+    }
+    readonly author?: {
+      readonly name: string
+    }
+  }>
+}
+interface ArtworkDetailsArticlesProps {
+  artwork: ArtworkDetailsArticlesArtwork
+}
+
+export class ArtworkDetailsArticles extends React.Component<
+  ArtworkDetailsArticlesProps
+> {
+  render() {
+    const { articles } = this.props.artwork
+    if (!articles || articles.length < 1) {
+      return null
+    }
+    return <Box>{articles.length}</Box>
+  }
+}
```

Comment:
@oxaudo - can you elaborate on why all of these interface props are here? If this whole component is only used to display a number it might be best to merge it into its parent. Otherwise, its best to limit the prop interface to what is explicitly being used inside of the component.